packages/expo-updates/ios/EXUpdates/EnabledAppController.swift

```diff
@@ -157,6 +159,30 @@ public class EnabledAppController: InternalAppControllerInterface, StartupProced
     stateMachine.queueExecution(stateMachineProcedure: procedure)
   }
 
+  // MARK: - UpdatesExternalMetricsInterface
+
+  // swiftlint:disable implicit_getter
+  public var runtimeVersion: String? {
+    get {
+      return config.runtimeVersion
+    }
+  }
+
+  public var updateURL: URL? {
+    get {
+      return config.updateUrl
+    }
+  }
+
+  public var launchedUpdateId: String? {
+    return getConstantsForModule().launchedUpdate?.updateId.uuidString
+  }
+
+  public var embeddedUpdateId: String? {
+    return getConstantsForModule().embeddedUpdate?.updateId.uuidString
+  }
```

Comment:
same comment as @tsapeta in last round about not calling `getConstantsForModule` if possible